Makefile

```diff
@@ -11,54 +11,24 @@ PIP = LDFLAGS="$(LDFLAGS)" pip
 WEBPACK = NODE_ENV=production ./node_modules/.bin/webpack
 
 develop: setup-git develop-only
-develop-only: clean setup-camunda node-version-check update-submodules install-yarn-pkgs install-clims-dev
+develop-only: clean node-version-check update-submodules install-yarn-pkgs install-clims-dev
 test: lint test-js test-python test-cli
 
 build: locale
 
-drop-db:
-	@echo "--> Dropping existing 'clims' database"
-	dropdb clims || true
-	@echo "--> Dropping existing 'test_clims' database"
-	dropdb test_clims || true
-
-create-camunda-tables:
-	# Assumes that the database is clean
-	@echo "--> Creating database tables for camunda"
-	./middleware/camunda/db-install.sh
-	./middleware/camunda/restart.sh
-
-CLIMS_DB_PASSWORD := $(shell openssl rand -base64 32)
-CLIMS_TEST_DB_PASSWORD := $(shell openssl rand -base64 32)
-
-setup-db-user: create-db
-	@echo "--> Make sure we have user called 'clims'"
-	-psql -d postgres -c "CREATE ROLE clims WITH LOGIN" || true
-	-psql -d postgres -c "ALTER ROLE clims WITH SUPERUSER;"
-	-psql -d clims -c "ALTER ROLE clims WITH LOGIN ENCRYPTED PASSWORD '$(CLIMS_DB_PASSWORD)'"
-
-	@echo "--> Make sure we have user called 'test_clims'"
-	-psql -d postgres -c "CREATE ROLE test_clims WITH LOGIN" || true
-	-psql -d postgres -c "ALTER ROLE test_clims WITH SUPERUSER;"
-	-psql -d clims -c "ALTER ROLE test_clims WITH LOGIN ENCRYPTED PASSWORD '$(CLIMS_TEST_DB_PASSWORD)'"
-
-	@echo "Add the following lines to ~/.pgpass:"
-	@echo "localhost:*:clims:clims:$(CLIMS_DB_PASSWORD)"
-	@echo "localhost:*:test_clims:test_clims:$(CLIMS_TEST_DB_PASSWORD)"
-	@echo "Set the permissions of ~/.pgpass to 0600: chmod 0600 ~/.pgpass "
-
-create-db:
-	@echo "--> Creating 'clims' database"
-	createdb -E utf-8 clims || true
+middleware:
+# Sets up all server middleware (postgresql, elasticsearch, redis etc.)
+	@./scripts/start-local-middleware.sh
+
+middleware-teardown:
+	@./scripts/stop-local-middleware.sh
 
 create-example-data:
 	@echo "--> Add example data"
 	lims createexampledata
 
-
-# TODO Don't know if we should create the user here, since it is not super intuative to reset the
-#      db to create a user. /JD 2019-10-03
```

Comment:
Removed this comment as it's now documented that for a fresh dev system one uses `make fresh`, which I guess must include having a test user. Since this is the first step everyone goes through it makes sense to me to have it there. Tell me what you think.